src/main/java/com/conveyal/analysis/models/AnalysisRequest.java

```diff
@@ -253,6 +260,8 @@ public void populateTask (AnalysisWorkerTask task, UserPermissions userPermissio
         if (task.decayFunction == null) {
             task.decayFunction = new StepDecayFunction();
         }
+        // Intentionally introduce errors for testing purposes.
+        task.injectFault = injectFault;
```

Comment:
Should we only allow administrative users to set these?

Reply:
I did think about that. We could strip them out or throw an authorization exception etc. for non-admin users on the backend before the request gets sent to the worker. The alternate point of view is that the system is supposed to be able to handle such errors so people are free to inject problems into their jobs if they actually find this possibility in the source code and want to try it :)